CMakeLists.txt

```diff
@@ -319,8 +319,8 @@ if (COMPILER_CLANG)
     endif()
 endif ()
 
-set (COMPILER_FLAGS "${COMPILER_FLAGS}")
-
+# Disable floating-point expression contraction in order to get consistent floating point calculation results across platforms
+set (COMPILER_FLAGS "${COMPILER_FLAGS} -ffp-contract=off")
```

Comment:
Please put in a newline after l. 323 to make this visually more pleasing, and I'll be happy :-)

Reply:
Done